lua/obsidian/command.lua

```diff
@@ -343,8 +338,41 @@ command.follow = function(client, _)
   if not note_name:match "%.md" then
     note_name = note_name .. ".md"
   end
+  return note_name
+end
+
+---Find backlinks for the note under the cursor
+---
+---@param client obsidian.Client
+command.cursor_backlinks = function(client, _)
+  local note_name = get_note_name()
+  if note_name == nil then
+    return
+  end
+
+  local ok, backlinks = pcall(function()
+    return require("obsidian.backlinks").new(client, note_name)
+  end)
```

Comment:
you mean in this case backlinks:view() will not show the note which has been referenced in the subdirectory, right?
but when we use `command.backlinks` I think will also encounter the issue.

Reply:
other comments were resolved, I'm just confused with this comment.

Reply:
Sorry, I don't think my link pointed to what I wanted. This is the correct link: https://github.com/epwalsh/obsidian.nvim/blob/1bc96d140022940473f3e174ad92a6f9bba13037/lua/obsidian/command.lua#L360-L378

In my case my vault is `~/epwalsh-notes`, but I put all of my markdown notes in a subdirectory: `~/epwalsh-notes/notes`. So currently your code will assume the path of my note is `~/epwalsh-notes/note.md`, when it's actually `~/epwalsh-notes/notes/note.md`, for example.

The backlinks command doesn't have this issue because it gets the full, unambiguous path to the current note for free.

Reply:
exactly, I've update the code to fix this